openassessment/xblock/resolve_dates.py

```diff
@@ -24,6 +24,18 @@ class DateValidationError(Exception):
 DISTANT_FUTURE = dt.datetime(dt.MAXYEAR, 1, 1, tzinfo=pytz.utc)
 
 
+def get_current_time_zone(open_assessment_block):
```

Comment:
It's usually better to pass in the granularity of the object that is actually needed by the function instead of a higher layer object.  In this case, it's not the ORA block that is really needed by this time_zone function.  Rather, it needs the runtime service in order to retrieve the user and their preferences.